plugins/status/plugin_test.go

```diff
@@ -480,3 +480,43 @@ func testStatus() *bundle.Status {
 
 	return &status
 }
+
+type testPlugin struct {
+	reqs []UpdateRequestV1
+}
+
+func (p *testPlugin) Start(context.Context) error {
```

Comment:
[nit] -- feel free to ignore
```suggestion
func (*testPlugin) Start(context.Context) error {
```

Reply:
nits are always appreciated, i'm still learning golang 😄